src/BartIO.jl

```diff
@@ -78,8 +141,8 @@ function read_cfl(filename::String)
     for i in eachindex(data)
         data[i] = read(fid, Float32) + 1im * read(fid, Float32)
     end
-
     close(fid)
+    data = reshape(data,dims...)
```

Comment:
Does this line do anything? `data = Array{ComplexF32}(undef, Tuple(dims))` already creates `data` in the right format, doesn't it?